core-bundle/src/DependencyInjection/Security/ContaoLoginFactory.php

```diff
@@ -53,19 +52,15 @@ protected function createAuthProvider(ContainerBuilder $container, $id, $config,
     {
         $twoFactorProviderId = TwoFactorFactory::PROVIDER_ID_PREFIX.$id;
         $twoFactorFirewallConfigId = 'contao.security.two_factor_firewall_config.'.$id;
-        $twoFactorFirewallConfig = new Definition(TwoFactorFirewallConfig::class, [$config, $id, null]);
 
         $container
-            ->setDefinition($twoFactorFirewallConfigId, $twoFactorFirewallConfig)
-            ->replaceArgument(2, new Reference('security.http_utils'))
+            ->setDefinition($twoFactorFirewallConfigId, new ChildDefinition(TwoFactorFirewallConfig::class))
```

Comment:
Shouldn't that be `new ChildDefinition(TwoFactorFactory::FIREWALL_CONFIG_DEFINITION_ID)`?

Reply:
Yes, obviously 🙈 Thanks!